changes/pr126.yaml

```
@@ -0,0 +1,2 @@
+feature:
```

```suggestion
enhancement:
```
as a rule of thumb we usually reserve `feature` for the big big stuff (e.g., the helm chart) and `enhancement` for the rest